### ISSUE TITLE
MaterialDesignFloatingHintTextBox in a dragablz control "leaves" dotted line when switching to other tab

### ISSUE THREAD

**Issue by tomh333** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1882]
When updating the material design themes package to v3 (3.1.3.1513), the MaterialDesign text boxes used in a Dragablz control "leave" a dotted line when switching to another tab. This problem did not occur with the exact same code when using material design v2.

**Reproduction:**

- Clone https://github.com/tomh333/TextBoxBugReproduction/
- Restore nuget packages
- Debug
- Switch Tabs by clicking on the "Tab 2" control 

**Problem**

When switching to another tab, the text boxes on the previous tab disappear, however they do leave a dotted line as seen in the screenshots below. They lines should not be visible when switching tabs.

- The first tab with the MaterialDesignFloatingHintTextBoxes:
![tab1](https://user-images.githubusercontent.com/34492153/83504753-29802d00-a4c5-11ea-85d6-b639d32cf675.png)

- The second tab on which the dotted lines at the location of the textboxes in tab 1 are visible:
![tab2](https://user-images.githubusercontent.com/34492153/83504441-cee6d100-a4c4-11ea-9f72-dec7ea1920a1.png)

**Comment by ApokalmiX** [comment]
I have the same problem since the update, a fix is to encapsulate the content of your tabItem with an AdornerDecorator as follows:

                         <TabItem Header = "My TabItem">
                             <AdornerDecorator>
                                 <! - My TabItem content ->
                             </AdornerDecorator>
                         </TabItem>

**Comment by w-syss** [comment]
This is the current (refactored) logic of the BottomDashedLineAdorner.
```
var element = (UIElement)d;
var adornerIsAttached = (bool)e.NewValue;

if (adornerIsAttached)
{
     element.AddAdorner(new BottomDashedLineAdorner(element));
}
else
{
     element.RemoveAdorners<BottomDashedLineAdorner>();
}
```

`e.NewValue` is true if the control was disabled and false if the control was disabled.
I think this should actually be reversed:

```
var element = (UIElement)d;
var controlWasDisabled = (bool)e.NewValue;

if (controlWasDisabled)
{
     element.RemoveAdorners<BottomDashedLineAdorner>();
}
else
{
     element.AddAdorner(new BottomDashedLineAdorner(element));
}
```

Before i submit this pull request however i wanted to ask @greuelpirat (the original author) if he has any objections to this.

**Comment by greuelpirat** [comment]
I cloned https://github.com/tomh333/TextBoxBugReproduction and could reproduce the issue.

I updated local `TextBoxBugReproduction` to use current head of MDIX and it seems that #1842 fixed that. This pull request added the required `AdornerDecorator` as @ApokalmiX suggested.

![image](https://user-images.githubusercontent.com/16866222/83540365-7f21fd00-a4f8-11ea-932c-7dd5e5cbc43b.png)

> `e.NewValue` is true if the control was disabled and false if the control was disabled.
> I think this should actually be reversed:

@w-syss I guess you want to refactor such bindings (get rid of `NotConverter`):

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/93d30d9b46a100688e914087100039bc5caff9a6/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml#L705

I see that this would reduce complexity. But this also means that the default value for this attached property must be `true`. Personally I avoid such properties. But after all I do not see any risks here.
